build/patch-dev-images.sh

```diff
@@ -4,22 +4,72 @@
+acm_installed_namespace=`oc get subscriptions.operators.coreos.com --all-namespaces | grep advanced-cluster-management | awk '{print $1}'`
+VERSION_TAG=${VERSION_TAG:-"latest"}
+DOCKER_URI="quay.io/stolostron"
+
+echo "* Patching hub cluster to ${VERSION_TAG}"
+oc annotate MultiClusterHub multiclusterhub -n ${acm_installed_namespace} mch-pause=true --overwrite
+
+# Patch the UI on the hub
+COMPONENT="grc-ui"
+LABEL="component=ocm-grcui"
+DEPLOYMENT=$(oc get deployment -l ${LABEL} -n ${acm_installed_namespace} -o=jsonpath='{.items[*].metadata.name}')
+oc patch deployment ${DEPLOYMENT} -n ${acm_installed_namespace} -p "{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"${COMPONENT}\",\"imagePullPolicy\":\"Always\",\"image\":\"${DOCKER_URI}/${COMPONENT}:${VERSION_TAG}\"}]}}}}"
+
+# Patch the API on the hub
+COMPONENT="grc-ui-api"
+LABEL="component=ocm-grcuiapi"
+DEPLOYMENT=$(oc get deployment -l ${LABEL} -n ${acm_installed_namespace} -o=jsonpath='{.items[*].metadata.name}')
+oc patch deployment ${DEPLOYMENT} -n ${acm_installed_namespace} -p "{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"${COMPONENT}\",\"imagePullPolicy\":\"Always\",\"image\":\"${DOCKER_URI}/${COMPONENT}:${VERSION_TAG}\"}]}}}}"
+
+# Patch the propagator on the hub
+COMPONENT="governance-policy-propagator"
+LABEL="component=ocm-policy-propagator"
+DEPLOYMENT=$(oc get deployment -l ${LABEL} -n ${acm_installed_namespace} -o=jsonpath='{.items[*].metadata.name}')
+oc patch deployment ${DEPLOYMENT} -n ${acm_installed_namespace} -p "{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"${COMPONENT}\",\"imagePullPolicy\":\"Always\",\"image\":\"${DOCKER_URI}/${COMPONENT}:${VERSION_TAG}\"}]}}}}"
+
+# Patch the addon-controller on the hub
+COMPONENT="governance-policy-addon-controller"
+LABEL="component=ocm-policy-addon-ctrl"
+DEPLOYMENT=$(oc get deployment -l ${LABEL} -n ${acm_installed_namespace} -o=jsonpath='{.items[*].metadata.name}')
+oc patch deployment ${DEPLOYMENT} -n ${acm_installed_namespace} -p "{\"spec\":{\"template\":{\"spec\":{\"containers\":[{\"name\":\"manager\",\"imagePullPolicy\":\"Always\",\"image\":\"${DOCKER_URI}/${COMPONENT}:${VERSION_TAG}\",\"env\":[{\"name\":\"CERT_POLICY_CONTROLLER_IMAGE\",\"value\":\"${DOCKER_URI}/cert-policy-controller:${VERSION_TAG}\"},{\"name\":\"IAM_POLICY_CONTROLLER_IMAGE\",\"value\":\"${DOCKER_URI}/iam-policy-controller:${VERSION_TAG}\"},{\"name\":\"CONFIG_POLICY_CONTROLLER_IMAGE\",\"value\":\"${DOCKER_URI}/config-policy-controller:${VERSION_TAG}\"},{\"name\":\"GOVERNANCE_POLICY_SPEC_SYNC_IMAGE\",\"value\":\"${DOCKER_URI}/governance-policy-spec-sync:${VERSION_TAG}\"},{\"name\":\"GOVERNANCE_POLICY_STATUS_SYNC_IMAGE\",\"value\":\"${DOCKER_URI}/governance-policy-status-sync:${VERSION_TAG}\"},{\"name\":\"GOVERNANCE_POLICY_TEMPLATE_SYNC_IMAGE\",\"value\":\"${DOCKER_URI}/governance-policy-template-sync:${VERSION_TAG}\"}]}]}}}}"
```

Comment:
This line is really long. Could we maybe put in line breaks, split this up into `oc set env` commands, or put this JSON in a file? Not necessary, but it would improve readability...

Reply:
I will do it afterwards. Can we get this merged first?

Reply:
Yes, that works for me.